webapp/channels/package.json

```diff
@@ -33,7 +33,7 @@
     "crypto-browserify": "3.12.0",
     "css-vars-ponyfill": "2.4.8",
     "date-fns": "2.29.3",
-    "dynamic-virtualized-list": "github:mattermost/dynamic-virtualized-list#3fe918b41de4cb08dbf43f1207bb58827b38e833",
+    "dynamic-virtualized-list": "github:Aryakoste/dynamic-virtualized-list#020a6fdcb481ca96a89c2904b30ed90918438692",
```

Comment:
We cannot use forks of libraries from contributors, for security reasons. Can you flip this back, and if you need to make changes to the library please create a PR on either the library itself, or on our fork.